clippy_lints/src/non_expressive_names.rs

```diff
@@ -199,6 +199,9 @@ impl<'a, 'tcx, 'b> SimilarNamesNameVisitor<'a, 'tcx, 'b> {
             );
             return;
         }
+        if interned_name.starts_with('_') {
```

Comment:
Can you add a comment saying why this is there?

Reply:
sure